src/commands/json2ts.ts

```diff
@@ -40,9 +41,14 @@ const generateAndWriteTsFile = async (
     .join('\n')
 
   const tsWithImports = `${imports ? `${imports}\n\n` : ''}${ts}`
+  const tsFormatted = await format(tsWithImports, {
```

Comment:
IMO instead of this we should provide to the user the ability to define his own formatting function.
As asked in another comment, if someone wants to use biome or oxc to format these files, it will be easier for them to do so.

Reply:
The config options get passed along to `json-schema-to-typescript`, which uses `prettier` internally to format the code it generates (see [here](https://github.com/bcherny/json-schema-to-typescript/blob/master/src/formatter.ts)). It doesn't support using other formatters. This fix just makes sure we format the generated code (imports included) in the same way.

Reply:
Thank for the clarification. Now I got it.